### PR TITLE
Support UWP

### PR DESCRIPTION
**Pull request by logangorence**
This pull request implements Universal Windows support.

### CODE REVIEW

**Comment by logangorence** [comment]
This also has some code that I started for determining whether WebSocket implementations actually support binary. It's not finished yet, but it doesn't affect anything.

File: DSLink.UWP.Example/DSLink.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+using System.Threading.Tasks;
+using ValueType = DSLink.Nodes.ValueType;
+
+namespace DSLink.UWP.Example
+{
+    public class ExampleDSLink : DSLinkContainer
+    {
+        public ExampleDSLink(Configuration config) : base(config)
+        {
+            Responder.AddNodeClass("testAction", delegate (Node node)
```
Comment (Pacane) [inline DSLink.UWP.Example/DSLink.cs]: why `AddNodeClass` and not `AddNode` ?

File: DSLink.UWP/UWPPlatform.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+using PCLStorage;
+
+namespace DSLink.UWP
+{
+    public class UWPPlatform
+    {
+        public static void Initialize()
+        {
+            Configuration.StorageBaseFolder = FileSystem.Current.LocalStorage;
+            Websockets.Universal.WebsocketConnection.Link();
```
Comment (Pacane) [inline DSLink.UWP/UWPPlatform.cs]: This looks weird. It looks like a method with a side effect, but there's no web in there?

Comment (logangorence) [inline DSLink.UWP/UWPPlatform.cs]: This sets up platform specific code for Universal Windows, and all other platforms have it as well.

Comment (Pacane) [inline DSLink.UWP/UWPPlatform.cs]: Ok, but still.. Is that your code or some 3rd party?

Comment (Pacane) [inline DSLink.UWP/UWPPlatform.cs]: Or maybe it's all right, I just never use "Link" as a verb, because well, I'm not natively speaking english.

**Comment by Pacane** [comment]
Approve.

**Comment by kaendfinger** [comment]
Approved

**Comment by Pacane** [comment]
APPROVED ALREADY

**Pull request merged**